app/build.gradle

```diff
@@ -2,11 +2,11 @@ apply plugin: 'com.android.application'
 
 android {
     compileSdkVersion 29
-    buildToolsVersion '28.0.3'
+    buildToolsVersion '30.0.2'
 
     defaultConfig {
         applicationId "com.bitrise_io.sample_apps_android_simple_google_play_deploy"
-        minSdkVersion 15
+        minSdkVersion 17
         targetSdkVersion 29
```

Comment:
Can/should we change the target to `30`?

Reply:
We can, Google Play accepts SDK 29 still for updates: https://support.google.com/googleplay/android-developer/answer/9859152?visit_id=637616936962637864-1730659907&rd=1#targetsdk&zippy=%2Ctarget-api-level-requirements-for-play-console
Should update sometime, including behavioral changes.